### ISSUE TITLE
Built error: "BREAKING CHANGE: It's no longer allowed to omit the '-loader' suffix when using loaders. You need to specify 'babel-loader' instead of 'babel'."

### ISSUE THREAD

**Issue by mattccrampton** [DominicTobias/universal-react#27]
```
> npm run build

universal-react@1.0.0-rc5 build /home/matt/code/universal-react
NODE_ENV=production webpack --config ./webpack.production.config.js --progress

Hash: 8b7ef33165a3b507df89
Version: webpack 2.2.1
Time: 58ms

ERROR in Entry module not found: Error: Can't resolve 'babel' in '/home/matt/code/universal-react'
BREAKING CHANGE: It's no longer allowed to omit the '-loader' suffix when using loaders.
                 You need to specify 'babel-loader' instead of 'babel'.

npm ERR! Linux 4.4.0-59-generic
npm ERR! argv "/home/matt/.nvm/versions/node/v4.3.2/bin/node" "/home/matt/.nvm/versions/node/v4.3.2/bin/npm" "run" "build"
npm ERR! node v4.3.2
npm ERR! npm  v2.14.12
npm ERR! code ELIFECYCLE
npm ERR! universal-react@1.0.0-rc5 build: `NODE_ENV=production webpack --config ./webpack.production.config.js --progress`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the universal-react@1.0.0-rc5 build script 'NODE_ENV=production webpack --config ./webpack.production.config.js --progress'.
npm ERR! This is most likely a problem with the universal-react package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     NODE_ENV=production webpack --config ./webpack.production.config.js --progress
npm ERR! You can get their info via:
npm ERR!     npm owner ls universal-react
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/matt/code/universal-react/npm-debug.log
```

**Comment by dominictobias** [comment]
Thanks I'll take a look at this

**Comment by dominictobias** [comment]
Fixed on master/1.0.0
